docs/standard/serverless-architecture/architecture-approaches/architecture-patterns.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Serverless apps. Architecture, patterns, and Azure implementation.
+description: Serverless apps. Architecture, patterns, and Azure implementation. | Architecture patterns
+keywords: Serverless, Microservices, .NET, Azure, Azure Functions
+author: JEREMYLIKNESS
+ms.author: jeliknes
+ms.date: 1/26/2018
+ms.prod: .net
+ms.technology: dotnet
+ms.topic: article
+---
+# Architecture patterns
```

Comment:
We've had similar content in the other eBooks as well. Can we link to it, and have one article that provides the survey of different architectures? /cc @CESARDELATORRE

Reply:
I tried to keep this high level and link out to more detailed resources, but happy to work with any additional feedback.

Reply:
I hate making comments like this one, because I don't have a recommendation 😞  As long as we keep thinking about it.